polygon-digital-carbon/src/utils/C3.ts

```diff
@@ -0,0 +1,22 @@
+import { BigInt } from '@graphprotocol/graph-ts'
+import { C3OffsetRequest } from '../../generated/schema'
+
+export function loadOrCreateC3OffsetBridgeRequest(requestId: string): C3OffsetRequest {
+  let request = C3OffsetRequest.load(requestId)
+
+  if (request == null) {
+    request = new C3OffsetRequest(requestId)
+    request.status = 'AWAITING'
```

Comment:
best practice for strings like this is to store them in a constant and then always reference it like `AWAITING`

not a big deal, just a nit pick

Reply:
created an assemblyscript enum. The enums from the actual schema aren't generated via codegen so I don't see an immediate way to mirror the enum but this at least creates more visibility & a single source to update if the schema enum changes.